python/mxnet/ndarray/ndarray.py

```diff
@@ -3740,3 +3740,32 @@ def empty(shape, ctx=None, dtype=None):
     if dtype is None:
         dtype = mx_real_t
     return NDArray(handle=_new_alloc_handle(shape, ctx, False, dtype))
+
+
+def histogram(a, bins=10, range_=None):
```

Comment:
Why underscore suffix for `range`?

Reply:
pylint complains if we override the keyword ``range``

Reply:
@piiswrong `numpy.histogram` has parameter `range`. Should we disable pylint check here to make `mx.nd.histogram` parameter naming consistent with numpy?